### PR TITLE
add task

### PR DESCRIPTION
**Pull request by IvanRamyk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,30 @@
...
+In this task, you should create a manager for the dataclass `CinemaHall`.
+
+#### 1. Create dataclass inside `models.py` module
+`CinemaHall` should have the following attributes:
+- id - unique identifier for each cinema hall
+- capacity - max number of visitors
+- description - general information about hall
+
+#### 2. Create database
+Create a database where will be stored entries with data about different cinema halls.
```
Comment (danylott) [inline README.md:14]: Add database name please: `cinema`

File: app/main.py [hunk 2]
```diff
@@ -1,3 +1,2 @@
-# TODO: add initial code
-def hello_world():
-    return "Hello, world!"
+if __name__ == '__main__':
+    pass
```
Comment (danylott) [inline app/main.py:2]: Here you can make the same, as was in example:
- import everything
- first line: CinemaHall.objects = CinemaHallManager()
- add some examples of how to call methods.
And make this code commented :)

Comment (danylott) [reply app/main.py:2]: Also delete tests folder

**Review by danylott — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -1,4 +1,30 @@
...
+Create a table `cinema_hall` with corresponding columns.
+Choose a descriptive name for your database.
+Now, you are ready to create a manager.
+
+#### 3. Create manager inside `managers.py` module
+Create `CinemaHallManager` class. 
+It should create a connection to the database inside the constructor.
+The manager should have the following methods:
```
Comment (danylott) [inline README.md:22]: add abbreviature `CRUD` here somehow

Comment (IvanRamyk) [reply README.md:22]: Done

File: README.md (hunk 1 above)
Comment (IvanRamyk) [reply README.md:14]: Done

File: app/main.py (hunk 2 above)
Comment (IvanRamyk) [reply app/main.py:2]: Done

**Review by danylott — APPROVED** [review]

**Review by danylott — APPROVED** [review]

**Pull request merged**